otwcode/otwarchive
# AO3-6078 Unescape title HTML when notifying user about work deletion

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6078 (please fill in issue number and remove this comment)

## Purpose

Show unescaped HTML for Title of fic in pop-up when deleted work

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?
Automated test was added

(If you have a Jira account with access, please update or comment on the issue
with any new or missing testing instructions.)

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*
Jonathan Means

(If you have a Jira account, please include the same name in the "Full name"
field on your Jira profile, so we can assign you the issues you're working on.)

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*
he/him

(If you do not fill in this section, we will use your GitHub account name and
they/them pronouns to avoid making assumptions about your gender.)

---

The HTML characters are still escaped in the email it sends. Not sure y'all's preference for that behavior, or if you want it included in this work if it is a bug.

Don't have permissions to assign myself stuff in JIRA, so just opening a PR and hopefully no one wastes too much time.
